README.md

```diff
@@ -76,6 +76,10 @@ Otherwise, installation of `modin[mpi]` may fail. Refer to
 [Installing with pip](https://unidist.readthedocs.io/en/latest/installation.html#installing-with-pip)
 section of the unidist documentation for more details about installation.
 
+**Note:** Since Modin 0.30.0 we use a reduced set of Ray dependencies: `ray` instead of `ray[default]`.
+This means that the dashboard and cluster launcher are no longer installed by default.
+If you need those, consider installing `ray[default]` (through pip) along with `modin[ray]` or `ray-default` (through conda).
```

Comment:
```suggestion
If you need those, consider installing `ray[default]` (through pip) or `ray-default` (through conda) along with `modin[ray]`.
```
The first suggestion above.
The second is maybe we should also add this note to installation with conda below like.

```
**Note:** Since Modin 0.30.0 we use a reduced set of Ray dependencies: `ray-core` instead of `ray-default`.
This means that the dashboard and cluster launcher are no longer installed by default.
If you need those, consider installing `ray-default` along with `modin-ray`.
```

Reply:
I would go with the second one.

Reply:
added